### PR TITLE
Code splitting and dynamic bundle loading by webpack

### PR DESCRIPTION
**Pull request by nickhsine**
@hanyulo @YuCJ @garfieldduck

### CODE REVIEW

**Comment by garfieldduck** [comment]
💯 🥇 :shipit:

File: webpack/dev.config.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports = {
   output: {
     path: assetsPath,
     filename: '[name]-[hash].js',
-    chunkFilename: '[name]-[chunkhash].js',
+    chunkFilename: '[id].chunk.js',
```
Comment (YuCJ) [inline webpack/dev.config.js]: add `hash` to prevent mis-cache

File: webpack/prod.config.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ module.exports = {
   output: {
     path: assetsPath,
     filename: '[name]-[chunkhash].js',
-    chunkFilename: '[name]-[chunkhash].js',
+    chunkFilename: '[id].chunk.js',
```
Comment (YuCJ) [inline webpack/prod.config.js]: add `hash` to prevent mis-cache

**Comment by YuCJ** [comment]
👍  💯

**Comment by YuCJ** [comment]
👍

**Pull request merged**